clay/gp.py

```diff
@@ -199,6 +199,26 @@ def from_data(cls, data, source, many=False):
 
         raise AssertionError()
 
+    def increment_playcount(self, callback=None):
+        """
+        Increments the gmusic playcount of the track (by one).
+
+        "callback" is called with "(track_id, error)" args after the
+        placount is incremented. (Optional)
+        """
+        if gp.is_subscribed:
+            track_id = self.store_id
+        else:
+            track_id = self.library_id
+
+        def on_playcount_incremented(track_id, error):
```

Comment:
I think this function call could be replaced by `lambda id_, error: callback(id_, error) if callback else None`

Reply:
You're right of course. 
Haven't been using python lately...

Reply:
No problem, that is what I am here for.

And it isn't like Python exactly advertises using lambdas either.